### ISSUE TITLE
Error while following "Hello world" example

### ISSUE THREAD

**Issue by Mister-Lemon** [Wouterdek/NodeNetwork#13]
Hi, 
I am following the instructions in ["Cookbook: Hello world in under 30 lines."](https://wouterdek.github.io/NodeNetwork/cookbook/hello_world). 

I am not sure what am I doing wrong, as Visual Studio mark this line as an error:
`        <nodenetwork:NetworkView x:Name="networkView" />`
On hover I see this error message:

> "Unable to cast object of type 'Splat.PlatformModeDetector' to type 'Splat.IModeDetector". 

Can you please assist? 

Thanks,
Noam

**Comment by Wouterdek** [comment]
Hmmm, it seems I can't reproduce this issue. Would you mind zipping your project folder and sharing it with me?

**Comment by Mister-Lemon** [comment]
Hi @Wouterdek , thanks for responding. 
I tried now to reproduce the issue myself. 
I opened new project and add NodeNetwork again via Nuget - and the issue is gone. everything works fine. 
Then I realized that when I had the issue, I accepted every updates suggested by Nuget for the depndecy packages. 
So I updated again all the packages, and now while I do not see the original error, I do have an exception while running the program:
```
An unhandled exception of type 'System.ArgumentException' occurred in System.Reactive.dll
Can't convert ReactiveUI.ReactiveCommand to System.Windows.Input.ICommand. To fix this, register a IBindingTypeConverter 
```

To summerize, I believe that ,my original issue can be closed as not reproducible. 
Just worth knowing that NodeNetwork is broken with the newer packages it depends on (I am not sure which one exactly). 

Reagrds,
Noam
